README.md

```diff
@@ -1,126 +1,158 @@
-# WelcomeBot Plugin (Beta) ![CircleCI branch](https://img.shields.io/circleci/project/github/mattermost/mattermost-plugin-welcomebot/master.svg)
+# Welcome Bot Plugin ![CircleCI branch](https://img.shields.io/circleci/project/github/mattermost/mattermost-plugin-welcomebot/master.svg)
 
-*TODO ADD DESC*
+Use this plugin to improve onboarding and HR processes. It adds a Welcome Bot that helps add new team members to channels.
 
-## Installation
+![image](https://user-images.githubusercontent.com/13119842/58736467-fd226400-83cb-11e9-827b-6bbe33d062ab.png)
 
-1. Go to the [releases page of this Github repository](https://github.com/mattermost/mattermost-plugin-welcomebot) and download the latest release for your Mattermost server.
-2. Upload this file in the Mattermost **System Console > Plugins > Management** page to install the plugin. To learn more about how to upload a plugin, [see the documentation](https://docs.mattermost.com/administration/plugins.html#plugin-uploads).
-3. Modify your `config.json` file to include the types of welcome messages you wish to send, under the `PluginSettings`. See below for an example of what this should look like.
+*Welcome a new team member to Mattermost Contributors team. Then add the user to a set of channels based on their selection.*
+
+![image](https://user-images.githubusercontent.com/13119842/58736540-30fd8980-83cc-11e9-8e8e-94ea3042b3b1.png)
+
+## Configuration
+
+1. Go to **System Console > Plugins > Management** and click **Enable** to enable the Welcome Bot plugin.
+    - If you are running Mattermost v5.11 or earlier, you must first go to the [releases page of this GitHub repository](https://github.com/mattermost/mattermost-plugin-welcomebot/releases), download the latest release, and upload it to your Mattermost instance [following this documentation](https://docs.mattermost.com/administration/plugins.html#plugin-uploads).
+
+2. Modify your `config.json` file to include your custom attributes, under the `PluginSettings`. See below for an example of what this should look like.
```

Comment:
1/5 `your custom attributes` is confusing. Maybe `your Welcome Bot's messages and actions`?

Reply:
Good catch, copy-paste error :)